src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1217,6 +1221,25 @@ public static void applySslContext()
         }
     }
 
+    public static void applyCryptoProvider()
+    {
+        try
+        {
+            if (conf.crypto_provider == null)
+                conf.crypto_provider = new ParameterizedClass("org.apache.cassandra.security.DefaultCryptoProvider", null);
+
+            Class<?> cryptoProviderClass = Class.forName(conf.crypto_provider.class_name);
```

Comment:
@ayushisingh29 what if `class_name` is null? Can't you have `conf.crypto_provider` non-null but class_name would be null? What happens if you have this in yaml?

    crypto_provider:
        - parameters:
            - a: "b"